### ISSUE TITLE
How do you add accessibilityRole and accessibilityStates?

### ISSUE THREAD

**Issue by fawcilize** [react-navigation/react-navigation#7365]
### Current behaviour

The bottom tab bar buttons are reading the accessibility label with VoiceOver or TalkBalk enabled.

### Expected behaviour

The bottom tab bar buttons should also read the accessibilityState and accessibilityRole. They could be overridable properties that are exposed, but ideally would have sane defaults.

### What have you tried

I've tried to override the tabBarButtonComponent property:
```
...

tabBarButtonComponent: TWPrimaryTabsButton

...

class TWPrimaryTabsButton extends React.Component {
  render() {
    const { onPress, onLongPress, testID, accessibilityLabel, ...props } = this.props;

    // Mostly a copy/pasta of the default component
    return (
      <TouchableWithoutFeedback
        onPress={onPress}
        onLongPress={onLongPress}
        testID={testID}
        hitSlop={{ left: 15, right: 15, top: 5, bottom: 5 }}
        accessibilityLabel={accessibilityLabel}
        accessibilityRole="imagebutton" // I added this
        accessibilityStates={['selected']} // I don't know how to calculate this. There is no focused prop or navigation state
      >
        <View {...props} />
      </TouchableWithoutFeedback>
    );
  }
}
```

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | both
| react-native          | 0.57.8
| react-navigation-tabs | 1.0.2
| node                  | v10.9.0
| npm           | 6.5.0
